api/build.go

```diff
@@ -514,11 +520,35 @@ func GetOrgBuilds(c *gin.Context) {
 	// nolint: gomnd // ignore magic number
 	perPage = util.MaxInt(1, util.MinInt(100, perPage))
 
+	privateRepos, err := database.FromContext(c).GetOrgPrivateRepoList(o)
+	if err != nil {
+		logrus.Errorf("unable to get private repos for org %s : %s", o, err)
```

Comment:
Should this return the error instead of moving on?

Reply:
I'll have it fail here, since the DB would most likely be the issue and it will fail on the next call as well.